src/ucp/tag/rndv.c

```diff
@@ -117,8 +117,10 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_rtr, (self),
     return status;
 }
 
-void ucp_tag_send_start_rndv(ucp_request_t *sreq)
+ucs_status_t ucp_proto_start_rndv(uct_pending_req_t *self)
 {
+    ucp_request_t *sreq = ucs_container_of(self, ucp_request_t, send.uct);
```

Comment:
It is possible that this function will be called a bunch of times (until ucp_rndv_send_rts returns success). Therefore the code in this function will be repeated as well, while there is no any need for that. I'd avoid that from performance perspective (even though it is  intended for RNDV messages).